docs/react-native/quick-setup.md

```diff
@@ -91,13 +88,11 @@ function StartStreamingButton({
 ### 3. Show other peers
 
 :::note
-
-In order to get peers, you must first join a room. See the steps above.
-
+In order to get peers, you must first join a room. See steps above.
```

Comment:
Can you restore these empty lines?

https://docusaurus.io/docs/markdown-features/admonitions#usage-with-prettier